ui/app/serializers/config-ui/message.js

```diff
@@ -34,9 +48,12 @@ export default class MessageSerializer extends ApplicationSerializer {
       href: json?.link_href || '',
     };
     // using the snapshot startTime and endTime since the json start and end times are null when
-    // it gets to the serialize function.
-    json.start_time = snapshot.record.startTime;
-    json.end_time = snapshot.record.endTime;
+    // it gets to the serialize function. we need to check to see if it's in iso format. if it's not
```

Comment:
These rows of comments are a little confusing. Can you clarify under what circumstances the serializer returns null (If I remember its if the values are coming from default values and have not been set by the user). 

Also, clarify the flow when we would be seeing a string vs data formatted object.

Let me know if that makes sense.

Reply:
Agreed that the comment is confusing! Let me try to refine it :)